### PR TITLE
637-wpcom_vip_get_term_by wpcom_vip_get_term_by is being deprecated.

### PR DESCRIPTION
**Pull request by pjpak**
Deprecating VIP Helper Functions wpcom_vip_get_term_by: https://lobby.vip.wordpress.com/2017/08/03/deprecating-vip-helper-functions/

### CODE REVIEW

File: php/datasource/class-fieldmanager-datasource-term.php [hunk 1]
```diff
@@ -164,8 +164,7 @@ public function presave_alter_values( Fieldmanager_Field $field, $values, $curre
 				}
 
 				// an affordance for our friends at WordPress.com
-				$term_by = function_exists( 'wpcom_vip_get_term_by' ) ? 'wpcom_vip_get_term_by' : 'get_term_by';
-				$term = call_user_func( $term_by, 'name', $value, $this->taxonomy );
+				$term = get_term_by( 'name', $value, $this->taxonomy );
```
Comment (dlh01) [inline php/datasource/class-fieldmanager-datasource-term.php]: The comment above this line is now outdated, too.

**Comment by pjpak** [comment]
Made recommended changes. thanks

**Comment by dlh01** [comment]
This looks 👍 to me. The Travis failure seems unrelated. I'll see whether I can fix that separately so Travis will pass on your branch.

**Comment by dlh01** [comment]
@pjpak Please merge `master` into your branch, which should fix the build errors in Travis.

**Comment by mboynes** [comment]
#645 merged this in. Thanks @pjpak!

**Pull request closed without merging**